packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useEffect, useState } from 'react';
+import { Text } from '@deriv/components';
+import { Notifications as Announcement } from '@deriv-com/ui';
+import { StandaloneBullhornRegularIcon } from '@deriv/quill-icons';
+import { load } from '@deriv/bot-skeleton';
+import { save_types } from '@deriv/bot-skeleton/src/constants/save-type';
+import clsx from 'clsx';
+import { localize } from '@deriv/translations';
+import AnnouncementDialog from './announcement-dialog';
+import { ANNOUNCEMENTS } from './config';
+import './announcements.scss';
+import { DBOT_TABS } from 'Constants/bot-contents';
+import { IconAnnounce, MessageAnnounce, TitleAnnounce } from './announcement-components';
+
+type TAnnouncements = {
+    is_mobile?: boolean;
+    handleTabChange: (item: number) => void;
+};
+
+const Announcements = ({ is_mobile, handleTabChange }: TAnnouncements) => {
+    const [isAnnounceDialogOpen, setIsAnnounceDialogOpen] = useState(false);
+    const [isOpenAnnounceList, setIsOpenAnnounceList] = React.useState(false);
+    const [amountAnnounce, setAmountAnnounce] = useState({} as Record<string, boolean>);
+    const buttonRef = React.useRef<HTMLButtonElement>(null);
+    const wrapperRef = React.useRef<HTMLDivElement>(null);
+    const accumulator_announcement = ANNOUNCEMENTS.ACCUMULATOR_ANNOUNCE;
+    const is_active_announce_1 = amountAnnounce?.announce_1;
+
+    const handleClickOutside = (event: React.MouseEvent<HTMLButtonElement>) => {
+        if (
+            !wrapperRef?.current?.contains(event.target as HTMLButtonElement) &&
+            !buttonRef?.current?.contains(event.target as HTMLButtonElement)
+        ) {
+            setIsOpenAnnounceList(false);
+        }
+    };
+
+    useEffect(() => {
+        document.addEventListener('click', handleClickOutside);
+        return () => {
+            document.removeEventListener('click', handleClickOutside);
+        };
+    }, []);
```

Comment:
We don't need to do this... for now, let's keep it, but let's reach out to the deriv-com/ui team and ask them to fix this from their side.


```suggestion
```